template/README.md

```diff
@@ -72,9 +70,8 @@ The `<[namespace]-[name]>` element should be used in situations where users may:
 
 Default [namespace]-[name]
 
-```html
-<[namespace]-[name]>Hello World</[namespace]-[name]>
-```
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./apiExamples/basic.md) -->
```

Comment:
```suggestion
<!-- AURO-GENERATED-CONTENT:START (CODE:src=./demo/apiExamples/basic.html) -->
```

Reply:
See below comment for why I don't want to move the apiExamples

Reply:
See below comment for why I don't want to move the apiExamples